src/main/resources/db/changelog-root.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd
+      http://www.liquibase.org/xml/ns/pro
+      http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd">
+<includeAll path="db/changelog/"/><!-- Automatically queries all files in this directory in lexical order -->
```

Comment:
why isn't this indented?
wouldnt the comment be better above or below, as long lines tend to get quite unreadable?

Reply:
```suggestion
   <!-- Automatically queries all files in this directory in lexical order -->
   <includeAll path="db/changelog/"/>
```